### DISCUSSION TITLE
[Feature] [Mobile] Prevent re-uploading pictures that have been deleted

### DISCUSSION THREAD

**Discussion by foux** [immich-app/immich#9727] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Right now, if we delete a picture from the web app, and this picture is still available on a device, it will be re-uploaded when launching Immich on that device.

Immich should restrain from re-uploading picture that have allready been sent (and deleted)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of: https://github.com/immich-app/immich/discussions/3594
